### DISCUSSION TITLE
v2.8.0版本中直播源导入链接http://y.qibaobaike.com/nzy.txt已失效

### DISCUSSION THREAD

**Discussion by SunshineboyZj** [Hunlongyu/ZY-Player#681] [建议]
**最新版安装后没有直播源，重置或重新安装软件也没有**
最新版本中直播源导入链接http://y.qibaobaike.com/nzy.txt已失效
请**更新导入链接**，谢谢
![屏幕截图 2021-04-05 040350](https://user-images.githubusercontent.com/81549837/113520209-21e95300-95c4-11eb-9a81-a26f6a355043.png)
![屏幕截图 2021-04-05 040404](https://user-images.githubusercontent.com/81549837/113520212-257cda00-95c4-11eb-9788-1d89d7c4b381.png)
![image](https://user-images.githubusercontent.com/81549837/115729054-a6efad00-a3b7-11eb-88bf-fed878aef19b.png)

**Comment by qishuai** [comment]
你好收到您的邮件。我会尽快给您回复。谢谢，保持联系！
